profile-widget/diveeventitem.cpp

```diff
@@ -194,7 +194,7 @@ void DiveEventItem::setupToolTipString(struct gasmix *lastgasmix)
 		}
 		*lastgasmix = *mix;
 	} else if (same_string(internalEvent->name, "modechange")) {
-		name += tr(": %1").arg(divemode_text[internalEvent->value]);
+		name += tr(": %1").arg(divemode_text_ui[internalEvent->value]);
```

Comment:
Doing `tr(": %1")` makes no sense to me. Not this string has to be translated but again the `divemode_text_ui` in the argument.